web/docs/data-model/prisma-file.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Prisma File
+---
+
+TODO: create a diagram showing how the Wasp file takes entities from the Prisma file
```

Comment:
Interesting I am curious to see what it turns out like!

Reply:
Nothing too fancy since it’s a simple relationship :D